spec/requests/api/services_spec.rb

```diff
@@ -445,6 +445,18 @@ def expect_svc_with_vms
       expect_result_resources_to_include_hrefs("resources", @svc1_vm_list)
     end
 
+    it "supports expansion of virtual attributes" do
+      run_get services_url, :expand => "resources", :attributes => "power_states"
+
+      expected = {
+        "resources" => [
+          a_hash_including("power_states" => %w(on on))
```

Comment:
I think these values come from the service's vms (defined somewhere above) - based on the default values from the factory. I think it could be a good idea to specify the raw power state of the vms when they are built, so they are independent of those default values